scripts/pre_push_hook.py

```diff
@@ -239,8 +256,8 @@ def compare_to_remote(remote, local_branch, remote_branch=None):
             to test against. If None same as local branch.
```

Comment:
If None, the remote branch is considered to be the same as the local branch.

Reply:
Done!